### PR TITLE
feat: /aws/fips redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1RIUejzxrQwvT-qQGG_FhRIaTVzZb07o6mZeiYkaBdqk/edit) and [figma](https://www.figma.com/design/FoWXkNxxdri9EgSsLdtTrH/24.10-AWS-bubble?node-id=34-1807&m=dev)
- Drive by:
  - I've implemented a different version of the WIP matrix mentioned [here](https://github.com/canonical/vanilla-framework/issues/5293) that works better with screen readers. If design approves, I'll update /azure/fips as well  

## QA

- View the site locally in your web browser at: https://ubuntu-com-14208.demos.haus/aws/fips
- Compare against doc and design 
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12839

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
Mostly looks good, just a few things to fix:

misaligned baselines - anchors outside paragraph? 
image

logo too big - should be 123x74
image


image needs replacing - can we please use this https://assets.ubuntu.com/v1/26c8a871-aws_workspaces.png

last section should have the deep p-section spacing:
image

File: templates/azure/sql.html [hunk 1]
```diff
@@ -46,64 +46,117 @@ <h1>SQL Server on Ubuntu Pro for Azure</h1>
   <section class="p-section">
     <div class="u-fixed-width p-section--shallow">
       <hr />
-      <h2>What’s inside SQL Server on Ubuntu Pro?</h2>
+      <h2>What's inside SQL Server on Ubuntu Pro?</h2>
```
Comment (mtruj013) [inline templates/azure/sql.html:49]: No content changes here, just a different implementation for this custom list which is more accessible (see pr description)

File: templates/aws/fips.html [hunk 2]
```diff
@@ -10,474 +10,513 @@
...
     <div class="row">
-      <div class="col-12">
-        <h2>What’s in Ubuntu Pro FIPS?</h2>
-        <ul class="p-matrix u-clearfix">
-          <li class="p-matrix__item">
-            <div class="p-matrix__content">
-              <h3 class="p-matrix__title">FIPS compliant by default</h3>
-              <div class="p-matrix__desc">
+      <div class="col-start-large-4 col-9 col-medium-6">
+        <hr class="is-muted" />
```
Comment (lizzochek) [inline templates/aws/fips.html:67]: Aren't we using p-rule--muted is such cases?

**Review by lizzochek — APPROVED** [review]
Except hr question, LGTM!

File: templates/aws/fips.html (hunk 2 above)
Comment (mtruj013) [reply templates/aws/fips.html:67]: We are indeed, thanks for flagging!

**Pull request merged**